src/ralph/security/models.py

```diff
@@ -82,6 +86,11 @@ class SecurityScan(
         on_delete=models.CASCADE,
     )
     vulnerabilities = models.ManyToManyField(Vulnerability, blank=True)
+    # this is a quirk field, it is updated manually (for now it's in API)
+    # this is because it's hard to handling it automatically
+    # (its value is computated depending on M2M field and M2M signals are
+    # complicated)
+    is_patched = models.BooleanField(default=False)
```

Comment:
I think it is.
https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.BooleanField

> If you need to accept null values then use NullBooleanField instead.